raster-foundry/raster-foundry
# Make logging configuration more consistent

## Overview

Changes necessary to more closely examine cache access patterns via DEBUG logs:

- Make `logbook.xml` configuration consistent
  - Standardize appender names and patterns
  - Add `com.azavea.rf` configuration, overridable `RF_LOG_LEVEL` where appropriate
- Remove timestamp from Nginx logging (already in Papertrail logs)
- Make default Memcached settings consistent with staging and production (timeout, local cache)
- Fix cache key creation (SHA-1 `Sets` and pull UUID values from `Some()`)
- Surface `RF_LOG_LEVEL` in `docker-compose.yml`

Fixes https://github.com/azavea/raster-foundry-platform/issues/412

### Checklist




## Testing Instructions

Modify the value of `RF_LOG_LEVEL` to `DEBUG` in `docker-compose.yml`. Then, spin up the local development environment with `server` and inspect the log output.

- Ensure that `DEBUG` level lots are emitted for the tile cache interactions 
- Ensure that modified cache keys are strings with no object syntax (`Some()`, `Set`)
- Ensure that timestamps are not present in Nginx logs
